utilities/ImpactUtilities.groovy

```diff
@@ -214,6 +232,15 @@ def updateCollection(changedFiles, deletedFiles, RepositoryClient repositoryClie
 		// files in a collection are stored as relative paths from a source directory
 		if (props.verbose) println "*** Deleting logical file for $file"
 		repositoryClient.deleteLogicalFile(props.applicationCollectionName, buildUtils.relativizePath(file))
+		repositoryClient.deleteLogicalFile(props.applicationOutputsCollectionName, buildUtils.relativizePath(file))
+	}
+
+	// remove renamed files from collection
+	renamedFiles.each { file ->
+		// files in a collection are stored as relative paths from a source directory
+		if (props.verbose) println "*** Deleting renamed logical file for $file"
+		repositoryClient.deleteLogicalFile(props.applicationCollectionName, buildUtils.relativizePath(file))
+		repositoryClient.deleteLogicalFile(props.applicationOutputsCollectionName, buildUtils.relativizePath(file))
```

Comment:
We should call buildUtils.relativizePath(file) one time in each block (performance best practices)